lib/Runtime/Language/JavascriptStackWalker.cpp

```diff
@@ -818,6 +825,7 @@ namespace Js
                 Assert(this->lastInternalFrameInfo.codeAddress);
                 this->lastInternalFrameInfo.frameConsumed = true;
             }
+#endif
```

Comment:
To make it more clear can you change this to `#endif // ENABLE_NATIVE_CODEGEN` and the above `#endif` to `#endif // DEBUG`?